node/rate_limiter.js

```diff
@@ -72,7 +75,18 @@ function RateLimiter(options) {
     }
     var self = this;
 
+    // stats
+    self.statEmitter = options.statEmitter || new StatEmitter();
```

Comment:
Since the code uses fast stats already, why do I need to use channel.buildStat()?